src/pages/index.js

```diff
@@ -1,58 +1,100 @@
+import Api from "../components/API.js";
 import Card from "../components/card.js";
 import Section from "../components/Section.js";
 import FormValidator from "../components/FormValidator.js";
 import UserInfo  from "../components/UserInfo.js";
-import { initialCards, validationSettings } from "../utils/constants.js";
+import { validationSettings } from "../utils/constants.js";
 import { PopupWithForm } from "../components/PopupWithForm.js";
```

Comment:
Please, delete `this._popupForm.reset();` from line 17 in `PopupWithForm`

You need to **reset** the form only   in a `then` block in order not to do that immediately after `submit` because when there is an error, the modal will still be opened and the user can try to send the data again to the server without having to type the data into the inputs once more.